### DISCUSSION TITLE
Difference between ImageMagick and ImageMagick-devel

### DISCUSSION THREAD

**Discussion by DOEHOONLEE** [ImageMagick/ImageMagick#4506] [General] [answered]
When installing ImageMagick, I found there are two types? of it. One is named `ImageMagick-devel` and its version seems to be 6.

What is the difference between ImageMagick and ImageMagick-devel??

![Screen Shot 2021-11-25 at 11 42 10 AM](https://user-images.githubusercontent.com/20305442/143370713-19849ade-5537-4ead-a5e1-f82d85a37807.png)

Here is the link to the CentOS pkg [CentOS pkg List](https://centos.pkgs.org/7/centos-x86_64/)

Thanks! As always, happy coding!

**Comment by snibgo** [comment, marked as answer]
I suppose the "devel" versions are for people who want to develop software that calls IM functions. I suggest that any new development use IM v7, not v6.

**Reply by DOEHOONLEE** [reply]
Oh, the ImageMagick-devel is for software development. There is no difference in use though, right? Except their versions are different.

![Screen Shot 2021-11-25 at 2 17 44 PM](https://user-images.githubusercontent.com/20305442/143383923-c2601893-d6c8-4487-a8af-ee55875508dc.png)

Install for linux on Download page introduces v6 only.

Is `git clone https://github.com/ImageMagick/ImageMagick.git ImageMagick-7.1.0` the only way to install v7 on Linux centOS/redhat?

Thank you for your answer!!

**Comment by fmw42** [comment]
I could be wrong, but I think the dev package contains the delegates for use with the normal version that has no delegates. I think on some systems you need both. But I will defer to others who know and use Linux.

**Comment by snibgo** [comment]
The IM version numbers are all 6.9.10-68. I don't know what "el7" means. "i686" and "x64_64" are different CPUs.

They will all have the same CLI (Command Line Interface).

I don't use Linux.
